R/utils_MCMC.R

```diff
@@ -270,6 +270,17 @@
 
   # Correction for those genes with total count = 0
   aux <- ifelse(aux == 0, min(aux[aux > 0]), aux)
+  
+  # Extra scaling if the data has spike-ins
+  if(length(altExpNames(Data)) > 0) {
```

Comment:
this assumes that if altExps exist then WithSpikes is true which isn't necessarily the case

Reply:
Solution is to add a with_spikes arg here and then add one also to BASiCS_PriorParam. Or to do a more detailed check here in release, and in devel to handle arguments properly

Reply:
Actually can get around this by setting mu.mu to be NULL in the PriorParam constructor and then setting it later based on the value of PriorParam$PriorMu, WithSpikes etc